packages/http-client-csharp/generator/TestProjects/CadlRanch/http/client/structure/multi-client/tspCodeModel.json

```diff
@@ -5,267 +5,69 @@
  "Enums": [
   {
    "$id": "2",
-   "kind": "enum",
-   "name": "ClientType",
-   "crossLanguageDefinitionId": "Client.Structure.Service.ClientType",
-   "valueType": {
+   "Kind": "enum",
+   "Name": "ClientType",
+   "CrossLanguageDefinitionId": "Client.Structure.Service.ClientType",
+   "ValueType": {
     "$id": "3",
-    "kind": "string",
-    "name": "string",
-    "crossLanguageDefinitionId": "TypeSpec.string",
-    "decorators": []
+    "Kind": "string",
+    "Name": "string",
+    "CrossLanguageDefinitionId": "TypeSpec.string",
```

Comment:
I think you need a merge from main, we recently changed the schema of tspCodeModel to make every property starting with a lower case letter to align from tcgc

Reply:
reverted